graphene/types/argument.py

```diff
@@ -8,6 +8,35 @@
 
 
 class Argument(MountedType):
+    """
+    Makes an Argument available on a Field in the GraphQL schema.
+
+    Arguments will be parsed and provided to resolver methods for fields as keyword arguments.
+
+    All ``arg`` and ``**extra_args`` for a ``graphene.Field`` are implicitly mounted as Argument
+    using the below parameters.
+
+    .. code:: python
+
+        age = graphene.String(
```

Comment:
Might want to run black on code snippets in docstrings here.

Reply:
age = graphene.String(
            # Boolean implicitly mounted as Argument
            dog_years=graphene.Boolean(description="convert to dog years"),
            # Boolean explicitly mounted as Argument
            decades=graphene.Argument(graphene.Boolean, default_value=False),
        )

Reply:
Should I omit `graphene.<Whatever>` like you did in your PR?

Reply:
I'd vote yes on this.

Reply:
Yes from me too